src/cpp/aiebu/src/elf/elfwriter.cpp

```diff
@@ -183,38 +183,38 @@ add_text_data_section(std::vector<writer>& mwriter, std::vector<symbol>& syms)
 {
   for(auto buffer : mwriter)
```

Comment:
This is a lot of copying,  should it be `auto&`

Reply:
We have the copy inefficiencies in several places. Will consider cleaning this up in a separate PR.